src/App.vue

```diff
@@ -90,13 +90,9 @@ export default {
 
     this.$store.dispatch('app/FETCH_VERIFIED_USERS')
 
-    EventBus.$on('socket:codex-coin:transferred', (codxCost) => {
```

Comment:
I moved these anonymous event handlers into component methods so they can be individually de-registered with `$off` below. If you just call `EventBus.$off('socket:codex-coin:transferred')`, that's going to deregister _all_ event handlers for that event (and not just the anonymous one you're adding here), which could cause problems if we ever register a listener somewhere else.